modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/tails.dm

```diff
@@ -1,3 +1,15 @@
+#define BETTER_VOX_TYPE "better_vox"
+#define FELINE_TYPE "feline"
+#define LIZARD_TYPE "lizard"
+#define VULPINE_TYPE "vulpine"
+#define MARINE_TYPE "marine"
+#define AVIAN_TYPE "avian"
+#define STRAIGHTTAIL_TYPE "straighttail"
+#define SHEPHERDLIKE_TYPE "shepherdlike"
+#define AXOLOTL_TYPE "axolotl"
+#define TESHARI_TYPE "teshari"
+#define STANDARD_TYPE "standard"
```

Comment:
I'd make these all into defines that follow the following structure instead, as it'll make it easier to find them:
`TAIL_TYPE_TYPENAMEHERE`
Also, is there really nowhere else in the code we check for the `general_type`? Not that it's necessarily a bad thing, just something I was wondering while you were there. It's been a long time since I touched this part of the code, in all honesty.